SU-SOE/stanford_soe_helper
# 7.x 2.x soe 2786 collections

# READY FOR REVIEW

# Summary
This includes various tweaks and fixes to the collections modules

# Needed By (Date)
- Sprint end

# Criticality
- Fixes broken stuff and tweaks things


# Steps to Test
- Switch to this branch  -  7.x-2.x-SOE-2786-Collections (https://github.com/SU-SOE/stanford_soe_helper/tree/7.x-2.x-SOE-2786-Collections)
- Enable all collections modules:
`drush en stanford_soe_helper_collection stanford_soe_helper_collection_administration stanford_soe_helper_collection_layouts stanford_soe_helper_collection_views`
- drush fr stanford_soe_helper_collection and drush fr stanford_soe_helper_collection_views
- For **SOE-2781**: Navigate to /node/add/stanford-collection and check to make sure that "Lead Text" is not required
- For **SOE-2796**: 
  -  Create a collection at: `/node/add/stanford-collection`
  - Navigate to `admin/manage/magazine_article` and add articles to the collection
  - Navigate to your collection and verify the articles appear as expected.
  - Navigate to an article and verify that your collection title appears above the article title
  - Edit your collection and unpublish the node.
  - Navigate back to your article and verify that your collection title *does not* appear about the article title.
- For **SOE-2797**: Navigate to an article that is in a collection that is published.
  - Go to the collection block at the bottom of the page and verify the current article page *does not* show up in this block.

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2781
https://stanfordits.atlassian.net/browse/SOE-2796
https://stanfordits.atlassian.net/browse/SOE-2797
## Related PRs

## More Information

## Folks to notify
@cjwest 


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
